server/api/models.py

```diff
@@ -26,11 +26,15 @@ class UserProfile(AbstractUser):
     pnl = models.IntegerField(default=0)
     portfolio = models.ManyToManyField(Property, related_name='portfolio', default=[], blank=True)
     watchlist = models.ManyToManyField(Property, related_name='watchlist', default=[], blank=True)
+    username = models.CharField(max_length=100, unique=True)
```

Comment:
Username is an inherent field in Django's AbstractUser Class. No need to add it separately